src/video_core/renderer_opengl/renderer_opengl.cpp

```diff
@@ -500,6 +508,15 @@ bool RendererOpenGL::Init() {
 
     return true;
 }
+void RendererOpenGL::FrameLimiter(float average_frame_time, uint32_t framelimit) {
+    // calculate  difference between average frame time and frame time
+    // for the framelimit, and sleep for the difference in time.
+    float frame_time = 1000.0f / framelimit;
+    if (average_frame_time < frame_time) {
+        std::this_thread::sleep_for(
+            std::chrono::milliseconds(long(frame_time - average_frame_time)));
```

Comment:
`FrameLimiter(average_frame_time, **std::chrono::seconds(1) / 60**);`
little problem there. std::chrono::seconds(1) / 60 will return 0 . had to use std::chrono::microsecond(1000000) / 60, which return 16667 us